timeless/sms.py

```diff
@@ -13,6 +15,28 @@ def send(self):
         raise NotImplementedError
 
 
+class RetrySendSMS:
+    """Decorator class for resending requests"""
+    def __init__(self, retry_count=5, timeout=1):
```

Comment:
@vladarefiev I'd say 3 retrials should be enough by default.

Reply:
@amihaiemil ok